hasher-matcher-actioner/terraform/collabs_example.json

```diff
@@ -9,6 +9,9 @@
             },
             "tags" : {
                 "L" : []
+            },
+            "active" : {
+                "S": "N"
```

Comment:
blocking: Can we make this type an AWS boolean value?https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBMapper.DataTypes.html

Reply:
sure will do